src/Output/Output.php

```diff
@@ -52,4 +57,13 @@ public function description(): string
     {
         return "";
     }
+
+    /**
+     * Sets the clock that should be used.
+     * @param Clock $clock Clock to use.
+     */
+    public function setClock(Clock $clock)

```

Comment:
The program runs as expected. The clock attribute is set default to null and the function getHolidayColor() creates a new object if null is passed.

I will change it that default a new clock object is build instead of null and rename the function to overrideClock.